### ISSUE TITLE
Add version numbers in apk names

### ISSUE THREAD

**Issue by RamPawar7** [FiorenMas/Revanced-And-Revanced-Extended-Non-Root#38]
Add the app version and patch version to the apk names so that we know if the available apks are up-to-date before downloading them.
Example: `youtube_armeabi-v7a_v19.44.39_revanced-extended_patch-v5.1.1.apk`

**Comment by FiorenMas** [comment]
Unnecessary, as my repository is already updated to the latest version with patches from RVE or RV within approximately 5-10 minutes. This feature is planned, but no ETA

**Comment by RamPawar7** [comment]
I asked this because today I wanted to download YT RVE armeabi v7a with the released 5.1.2 patches. First I downloaded the light version. After installing I found out that it was not updated (this was few hours after patch release). Then I downloaded regular version and it was updated.

That's why I ask.

**Comment by RamPawar7** [comment]
I asked this because today I wanted to download YT RVE armeabi v7a with the released 5.1.2 patches. First I downloaded the light version. After installing I found out that it was not updated (this was few hours after patch release). Then I downloaded regular version and it was updated.

That's why I ask.

**Comment by RamPawar7** [comment]
I asked this because today I wanted to download YT RVE armeabi v7a with the released 5.1.2 patches. First I downloaded the light version. After installing I found out that it was not updated (this was few hours after patch release). Then I downloaded regular version and it was updated.

That's why I ask.

I think you forgot to update the lite version

**Comment by FiorenMas** [comment]
YT lite RVE can't patch with lastest patches, because this patches support youtube version 19.44.39, this version didn't have split apk in apkmirror so can't make lite version.

**Comment by RamPawar7** [comment]
Also not all apps are updated after patch release. So how do we know which apps to download and update?

Atleast add the date and time to each app when they are updated so we know that an update is available for our app

**Comment by bagarwa** [comment]
I'll add one problem with there being only 1 release. 

I had the patched tiktok installed from here. That was working fine. Then the apk was updated recently, so I downloaded that and updated my tiktok installation. But this latest update has some issue and tiktok now crashes at launch. And now I don't have any way to go back. 

Since there's only 1 release, only the latest apk is available here and I have no way of getting to the previously working apk.

**Comment by FiorenMas** [comment]
> I'll add one problem with there being only 1 release.
> 
> I had the patched tiktok installed from here. That was working fine. Then the apk was updated recently, so I downloaded that and updated my tiktok installation. But this latest update has some issue and tiktok now crashes at launch. And now I don't have any way to go back.
> 
> Since there's only 1 release, only the latest apk is available here and I have no way of getting to the previously working apk.

Tiktok crash on startup because lastest rv patches, I revert old patch everything working fine now, and bump version tiktok to avoid tiktok warn old version
https://github.com/FiorenMas/Revanced-And-Revanced-Extended-Non-Root/commit/354b75bbb0dc0d8c3eceda9e50f3ba93aa7c444d

**Comment by bagarwa** [comment]
> The latest Revanced patches (v5.8.0) caused TikTok to crash, so I reverted to Revanced patches v5.7.2. Everything is working fine now. I also bumped the TikTok version to avoid TikTok's warning about an old version. See more: [354b75b](https://github.com/FiorenMas/Revanced-And-Revanced-Extended-Non-Root/commit/354b75bbb0dc0d8c3eceda9e50f3ba93aa7c444d)

Thanks for the quick fix. But you see the problem I was trying to highlight, right? This fix needed your intervention. If the release section had a releases history, I could've just gone back to previous release, downloaded the older apk and installed that, without needing to wait for you to upload a fixed apk.

**Comment by BR4CKUD3R** [comment]
According to your statement, beta utilizes the most recent versions of apk available, though there's no way to know until you install the app or analyze the apk your own work. In my instance, I utilize obtaninum to keep my applications up to date; nonetheless, apkmirror shows that the currently installed (facebook) version is much older than the version available on apkmirror. So having an apk version number will be quite useful in this for instance. There may be an obligation that you do not want to include the version number in the release, but you can always include the final four numbers, i.e., 0.66.49?

**Comment by skupjoe** [comment]
Agreed! Please support this thank you. I was just going to come in and request this.

This would help a lot in keeping track of downloaded apks and a system could be developed to keep things up-to-date with this type of tracking

**Comment by mJNayim** [comment]
It would be great to have this to update using Obtainium.

**Comment by anantoarefin** [comment]
It has been a while is there any ETA now for adding version/patch numbers for different app releases. I found there were new versions of Youtube and Yotube Music by andea on mobilism but they aren't the latest version. So those newer version are recommended by the revanced managers ig.

**Comment by nullcpy** [comment]
> app version is much needed because it's hard to understand which version is current

my builder builds with app version 
https://github.com/nullcpy/rvb

**Comment by anantoarefin** [comment]
Title Format: [Title]  [Patch Repository] [Architecture] [App Version] [Patch version]

Keep the tile short and precise so it's easily understandable 
YtM for YouTube Music etc

YtM Morphe Arm64 v(x.x.x) Pv(y.y.y)

**Comment by TheJNXx** [comment]
This would be very useful for Obtainium; there's no way for it to know when an update is available or not...

**Comment by coolswapnil** [comment]
An app version is much needed because it is hard to understand which version is current. This will also help ObtainX track the app version from the asset name. We will have more control and clarity over the app version installation.

**Comment by victay44** [comment]
> This would be very useful for Obtainium; there's no way for it to know when an update is available or not...

You can enable the option "use latest release assets" inside Obtainium. I can use it without issues.
